tests/core/editable/insertion.js

```diff
@@ -159,7 +159,79 @@
 			editor.insertHtml( '<p>bam</p>', 'html', range );
 
 			assert.isInnerHtmlMatching( '<p id="p1">foo</p><p>bam^@</p><p>bar@</p>',
-				bender.tools.selection.getWithHtml( editor ), { compareSelection: true, normalizeSelection: true } );
+				tools.selection.getWithHtml( editor ), { compareSelection: true, normalizeSelection: true } );
 		}
-	} );
+	};
+
+	// Add tests for (#2813).
+	addInsertionTests( [
```

Comment:
Small duplication here. I would suggest deciding between removing the comment (except tag reference) or inlining function.